apps/src/code-studio/pd/application_dashboard/cohort_view_table.jsx

```diff
@@ -90,6 +95,16 @@ export default class CohortViewTable extends React.Component {
         header: {
           label: 'Status',
           transforms: [sortable]
+        },
+        cell: {
+          format: (status) => {
+            return _.upperFirst(status);
+          },
+          transforms: [
+            (status) => ({
+              style: {...styles.statusCellCommon, ...styles.statusCell[status]}
+            })
```

Comment:
slick. That was nice thinking back when we exported `StatusColors` from `constants.js`